src/components/DatePicker/DatePicker.driver.ts

```diff
@@ -0,0 +1,159 @@
+    /**
+     * Click on the Nth day in the current displayed month
+     * @param n - the day index
+     */
+    clickOnNthDay: async (n: number) => WSRCalendarDriver.clickOnNthDay(n),
+    /**
+     * Click on a given day
+     * @param date - a date to select
+     */
+    clickDay: async (date: Date) => WSRCalendarDriver.clickDay(date),
+    /**
+     * Check if the given day is active
+     * @param date - a date
+     */
+    isDayActive: async (date: Date) => WSRCalendarDriver.isDayActive(date),
+    /**
+     * Click on the year dropdown
+     */
+    clickOnYearDropdown: async () => WSRCalendarDriver.clickOnYearDropdown(),
+    /**
+     * Click on the month dropdown
+     */
+    clickOnMonthDropdown: async () => WSRCalendarDriver.clickOnMonthDropdown(),
+    /**
+     * Click on the Nth dropdown option in the year dropdown
+     * @param n - the index of the year in the list
+     */
+    clickOnNthYear: async (n: number) => WSRCalendarDriver.clickOnNthYear(n),
+    /**
+     * Click on the prev month button
+     */
+    clickOnPrevMonthButton: async () =>
+      WSRCalendarDriver.clickOnPrevMonthButton(),
+    /**
+     * Click on the next month button
+     */
+    clickOnNextMonthButton: async () =>
+      WSRCalendarDriver.clickOnNextMonthButton(),
+    /**
+     * Check whether the year dropdown exist
+     */
+    isYearDropdownExists: async () => WSRCalendarDriver.isYearDropdownExists(),
+    /**
+     * Check whether the year caption exist
+     */
+    isYearCaptionExists: async () => WSRCalendarDriver.isYearCaptionExists(),
+    /**
+     * Check whether the month dropdown exist
+     */
+    isMonthDropdownExists: async () =>
+      WSRCalendarDriver.isMonthDropdownExists(),
+    /**
+     * Check whether the month caption exist
+     */
+    isMonthCaptionExists: async () => WSRCalendarDriver.isMonthCaptionExists(),
+    /**
+     * Get the month caption text
+     */
+    getMonthCaption: async () => WSRCalendarDriver.getMonthCaption(),
+    /**
+     * Get the label of the selected month dropdown option
+     */
+    getMonthDropdownLabel: async () =>
+      WSRCalendarDriver.getMonthDropdownLabel(),
+    /**
+     * Get the label of the selected year dropdown option
+     */
+    getSelectedYear: async () => WSRCalendarDriver.getSelectedYear(),
+    /**
+     *  Returns the text of the focused day or `null` if there is no focused day
+     */
+    getFocusedDay: async () => WSRCalendarDriver.getFocusedDay(),
+    /**
+     * Press left arrow
+     */
+    pressLeftArrow: async () => WSRCalendarDriver.pressLeftArrow(),
+    /**
+     * Press right arrow
+     */
+    pressRightArrow: async () => WSRCalendarDriver.pressRightArrow(),
+    /**
+     * Get the selected day
+     */
+    getSelectedDay: async () => WSRCalendarDriver.getSelectedDay(),
+    /**
+     * Trigger key down event
+     * @param key
+     */
+    triggerKeyDown: async key => WSRCalendarDriver.triggerKeyDown({ key }),
```

Comment:
I'm not sure I understand what `triggerKeyDown` is for 🤔

Reply:
It's in order to simulate a keyboard key down event. But I will remove it.
I also wasn't sure we need it..